### PR TITLE
Update examples with new builder methods to replace obsolete members

### PR DESCRIPTION
**Pull request by Shane32**
Fixes issues:
- #88 
- `AspNetCoreCustom` does not have working sample GraphiQL interface

### CODE REVIEW

**Comment by Shane32** [comment]
Tried this query with the `AspNetCore` project too:

```gql
query { hero { id } }
```

No console messages.  Seems to work fine for me.

image

**Comment by develorem** [comment]
I can confirm that when using the \ui\playground that it goes all white and a console error is logged:
`Cannot read properties of undefined (reading 'resolvers')`
Switching back to master and running again, do not experience either issue.
Using Visual Studio 2022, .NET 6.0.2 and Chrome Version 98.0.4758.102 (Official Build) (64-bit)
Note: It works in Firefox!

**Comment by Shane32** [comment]
Well, then I'd suggest clearing cache and cookies and all that stuff.  One thing that catches our team often is: if you have any CORS plugins, disable the plugin, but of course CORS shouldn't even come into play here.  Playground issues are outside of our control in any case, especially if it can be demonstrated that it works properly on other systems and/or other browsers.

**Comment by Shane32** [comment]
Note that I have Visual Studio script debugging disabled on my machine.  Perhaps you might check that it is disabled also on yours.

**Comment by Shane32** [comment]
Also of interest, I did not touch the line of code that installs playground:

```cs
app.UseGraphQLPlayground();
```

Nor has the nuget package changed or any of the code behind that method for the `master` branch compared to this branch.  I would probably investigate if playground is properly sending a request, and if it is receiving the same response (checking response properties like content-type) either way.  If the response is identical, it should not behave different.  If the response is different, which is possible, we would need to know what changed in order to fix it.  Like if the content-type header is wrong, then we can address it.  But without being able to reproduce the problem here, I won't be able to diagnose it further without your help unfortunately.

**Comment by Shane32** [comment]
I did notice that neither the AspNetCore nor the AspNetCoreMulti solutions return apollo metrics.  This could be a bug in the server project.  It was not working before this PR either, and does work in the AspNetCoreCustom project.

**Comment by develorem** [comment]
Clearing everything didn't help. But the fact that it works in firefox, and the fact that the solution with React front end in it works (and that I can see the response message does contain the graphql response in correct format) I'd be happy with this change to proceed.

**Comment by develorem** [comment]
Oh do you still need this reference?
    <PackageReference Include="GraphQL.Server.Transports.AspNetCore.SystemTextJson" Version="5.2.0" />

**Comment by Shane32** [comment]
> Oh do you still need this reference? `PackageReference Include="GraphQL.Server.Transports.AspNetCore.SystemTextJson" Version="5.2.0"`

Yes, for the call to `AddServer()`

File: src/AspNetCoreCustom/Example/public/index.html [hunk 1]
```diff
@@ -1,12 +1,78 @@
+<!--
```
Comment (sungam3r) [inline src/AspNetCoreCustom/Example/public/index.html:1]: Copied from server?

**Review by sungam3r — APPROVED** [review]

File: src/AspNetCoreCustom/Example/public/index.html (hunk 1 above)
Comment (Shane32) [reply src/AspNetCoreCustom/Example/public/index.html:1]: Copied from their example repo. Edited the url of the license file and GraphQL path

**Pull request merged**